front/app/containers/HomePage/SignedInHeader/HeaderImage.tsx

```diff
@@ -60,19 +51,17 @@ const HeaderImage = () => {
 
     return (
       <HeaderImageContainer>
-        <HeaderImageContainerInner data-cy="e2e-fixed-ratio-header-image-parent">
+        -{' '}
+        <HeaderImageContainerInner data-cy="e2e-header-image-parent">
           {/*
-            With this fixed ratio layout, the image would be messed up.
+            With the fixed ratio layout, the image would be messed up, so
             Ticket: https://citizenlab.atlassian.net/browse/CL-2215
           */}
           {tenantHeaderImage && !isFixedBannerLayout && (
             <StyledImage
-              data-cy="e2e-fixed-ratio-header-image"
+              data-cy="e2e-header-image"
```

Comment:
These are not uniquely for the fixed-ratio layout so I thought the names were confusing.